diaphora_ida.py

```diff
@@ -2806,6 +2845,51 @@ def main():
       os.remove("%s-crash" % file_out)
 
     idaapi.qexit(0)
+  # EXPORT - works with pseudocode, better than above
+  elif os.getenv("DIAPHORA_AUTO2") is not None:
+    if g_debug:
+      log("Handling DIAPHORA_AUTO2")
+      log("DIAPHORA_EXPORT_FILE=%s" % os.getenv("DIAPHORA_EXPORT_FILE"))
+    file_out = os.getenv("DIAPHORA_EXPORT_FILE")
+    if file_out is None:
+      raise Exception("No export file specified!")
+    _diff_or_export(False, file_out=file_out)
```

Comment:
While this method works, it misses many other features (like python scripts). But, well, it's acceptable.

Reply:
Can you explain more? I am not sure to see what you mean...

Reply:
Of course! So, for example, one can supply a python script for the current project in order to modify the export or diffing process. This code handles such an option:

```    bd = CIDABinDiff(file_out)
    project_script = os.getenv("DIAPHORA_PROJECT_SCRIPT")
    if project_script is not None:
      bd.project_script = project_script
    bd.use_decompiler_always = use_decompiler
```

In your case, it won't handle such options as they aren't handled inside `_diff_or_export()`. May I ask why did you add a new export method?

Reply:
OK gotcha, my bad. I've just re-tested with the existing code and indeed it works so will remove my code for this since indeed yours is more complete.